scripts/install/createDatasource.php

```diff
@@ -18,23 +18,38 @@
  *               
  */
 
+namespace oat\taoThemingPlatform\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\oatbox\filesystem\FileSystemService;
 use oat\taoThemingPlatform\model\PlatformThemingService;
 
-/*
- * This post-installation script creates a data storage directory
- * for the taoThemingPlatform extension.
+/**
+ * Class createDatasource
+ *
+ * @author Gyula Szucs, <gyula@taotesting.com>
  */
-$dataPath = FILES_PATH . 'taoThemingPlatform' . DIRECTORY_SEPARATOR;
-if (file_exists($dataPath)) {
-    helpers_File::emptyDirectory($dataPath);
-}
+class createDatasource extends InstallAction
+{
+    /**
+     * @param array $params
+     * @return \common_report_Report
+     */
+    public function __invoke($params)
+    {
+
+        /** @var FileSystemService $fsService */
+        $fsService = $this->getServiceManager()->get(FileSystemService::SERVICE_ID);
+        $fs = $fsService->createFileSystem(PlatformThemingService::FILE_SYSTEM_ID, 'taoThemingPlatform');
```

Comment:
Could u check if filesystem already exists to avoid to erase it in case of taoSetup